### ISSUE TITLE
Problem with fullscreen button in v12.0.0

### ISSUE THREAD

**Issue by valkoasd** [PierfrancescoSoffritti/android-youtube-player#957]
If i use default controls (**DefaultPlayerUiController**) for YouTubePlayerView, the fullscreen button does not trigger methods of **FullscreenListener**. When i checked in the code of **DefaultPlayerUiController** i found this:
```
init {
    onFullscreenButtonListener = View.OnClickListener {
      isMatchParent = !isMatchParent
      when (isMatchParent) {
        true -> youTubePlayerView.matchParent()
        false -> youTubePlayerView.wrapContent()
      }
    }

    onMenuButtonClickListener = View.OnClickListener { youTubePlayerMenu.show(menuButton) }

    initClickListeners()
  }
```
As you can see the button for fullscreen only changes the size of the player.

**Comment by PierfrancescoSoffritti** [comment]
Yes, that is intentional, [see doc](https://github.com/PierfrancescoSoffritti/android-youtube-player#full-screen). I suggest not using the custom UI unless you really need it.

**Comment by valkoasd** [comment]
If i decide to use **DefaultPlayerUiController** how can i be notified if the user was pressed the fullscreen button? I appologise for my misunderstanding

**Comment by PierfrancescoSoffritti** [comment]
Good point, I think [DefaultPlayerUIController](https://github.com/PierfrancescoSoffritti/android-youtube-player/blob/master/custom-ui/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/customui/DefaultPlayerUiController.kt) should expose full screen click listeners. I will add it at some point.

In the menatime you can probably get a reference to the fullscreen button and put a click listener on it. `findViewById(R.id.fullscreen_button)`.

**Comment by leah-morada-mediacorp** [comment]
Hi, is this feature added?

**Comment by PierfrancescoSoffritti** [comment]
Not yet.
